packages/confirm-dialog/test/confirm-dialog.test.js

```diff
@@ -1,8 +1,9 @@
 import { expect } from '@esm-bundle/chai';
-import { esc, fixtureSync, nextRender, oneEvent } from '@vaadin/testing-helpers';
+import { aTimeout, esc, fixtureSync, nextRender, oneEvent } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
 import './not-animated-styles.js';
 import '../vaadin-confirm-dialog.js';
+import { getDeepActiveElement } from '@vaadin/a11y-base/src/focus-utils.js';
```

Comment:
Looks like you need to add `@vaadin/a11y-base` dev dependency.

Reply:
Good point, added.